fspiop/v2_0_ISO20022/openapi3/components/schemas/Number.yaml

```diff
@@ -1,4 +1,3 @@
 title: Number
 type: string
-pattern: ^[\d\-\.\,]{1,18}$
```

Comment:
^(?!0\d)\d{1,18}$

I don't think we should support the negative.
Not sure why it is deleted.